### PR TITLE
Update details summary report, #1049

### PR DESCRIPTION
**Pull request by njkim**
For the resource models, Observation, Group and Place.

### CODE REVIEW

File: afs/templates/views/components/reports/place.htm [hunk 1]
```diff
@@ -120,6 +124,16 @@ <h1 class="afs-summary-report-title"><span data-bind="text: displayname" class="
...
                 </div>
                 <!-- /ko -->
+            </div>
+        </div>
+        <!-- Geospatial Definition -->
+        <div class="afs-report-section">
+            <h2 class="afs-report-section-title">{% trans "Geospatial Definitions" %}</h2>
+            <div class="afs-report-subsection" >
+                <!-- ko if: geojson == '--' -->
+                <h3 class="afs-report-nodata-message">{% trans "There's no geospatial definition for this resource" %}</h3>
```
Comment (aarongundel) [inline afs/templates/views/components/reports/place.htm:134]: For consistency, let's not use contractions.  So "There is no geospatial definition for this resource."

File: afs/templates/views/components/reports/observation.htm [hunk 2]
```diff
@@ -210,57 +222,57 @@ <h1 class="afs-summary-report-title"><span data-bind="text: displayname" class="
...
-                            <tr>
-                                <td><a class="afs-table-link" data-bind="click: function() {window.open(link);}, text: displayValue" href="javascript:void(0)"></a></td>
-                            </tr>
-                        </tbody>
-                    </table>
+            </div>
+        </div>
+        <!-- Person Carried out -->
+        <div class="afs-report-section">
+            <h2 class="afs-report-section-title">{% trans "Persons Carried out" %}</h2>
```
Comment (aarongundel) [inline afs/templates/views/components/reports/observation.htm:257]: This should be "Persons Carrying Out Observation" (remove s if this is cardinality 1)

**Review by aarongundel — CHANGES_REQUESTED** [review]
minor nits

File: afs/templates/views/components/reports/place.htm (hunk 1 above)
Comment (njkim) [reply afs/templates/views/components/reports/place.htm]: Fixed.

File: afs/templates/views/components/reports/observation.htm (hunk 2 above)
Comment (njkim) [reply afs/templates/views/components/reports/observation.htm]: Thanks!

**Review by aarongundel — APPROVED** [review]
lgtm

**Pull request merged**